src/App.jsx

```diff
@@ -14,45 +18,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABETICALLY = 'sort alphabetically';
+const SORT_FIELD_BY_LENGTH = 'sort by length';
+
+function getPreparedGoods(goods, sortField, isReversed) {
+  let preparedGoods = goods.map((good, index) => (
+    {
+      name: good,
+      id: index + 1,
+    }
+  ));
```

Comment:
```suggestion
  const preparedGoods = goods.map((good, index) => ({
      name: good,
      id: index + 1,
    }));
```
Prefer const over let, as sort and reverse mutate an array you don't need to rewrite the result into the variable